src/main/java/org/ibp/api/java/impl/middleware/inventory/manager/LotServiceImpl.java

```diff
@@ -113,7 +113,12 @@
 
 	@Override
 	public List<ExtendedLotDto> searchLots(final LotsSearchDto lotsSearchDto, final Pageable pageable) {
-		return this.lotService.searchLots(lotsSearchDto, pageable);
+		return this.searchLots(lotsSearchDto, pageable, false);
+	}
+
+	@Override
+	public List<ExtendedLotDto> searchLots(final LotsSearchDto lotsSearchDto, final Pageable pageable, final boolean applyResultsLimit) {
```

Comment:
Seems like this function is always calles with applyResultsLimit = true. Should not be better to rename the function and remove the boolean param? assuming that it will always be limited?

Reply:
Done!